cgi-bin/DW/External/Site.pm

```diff
@@ -96,7 +130,8 @@ sub get_site {
                 map { lc $_ }
                 split( /\./, $site );
 
-    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
+    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
```

Comment:
Oh, sure, okay - I'll remove those then.

The reason I didn't include an issue number is that to the best of my knowledge there isn't an open issue for this - I discussed it with folk in IRC, we agreed it would be nice to have, but I didn't ever file an issue. Let me know if you'd like me to file one retroactively?